tika-bundle/pom.xml

```diff
@@ -35,6 +35,7 @@
 
     <properties>
         <tika.version>1.18</tika.version>
+        <jackson.version>2.9.10</jackson.version>
```

Comment:
❓ jackson is also used in javalin. Should be be upgraded there too?
https://github.com/codice/thirdparty/blob/b1cc66dfe7451946c067bee6a3bab7ebdef7c091/javalin/pom.xml#L26

Reply:
This isn't used in DDF as far as I can tell, so I'd hesitate to update this without knowing how to test it with whatever projects do use this artifact. Since it doesn't affect DDF (if it does please point me to where it is used) this probably shouldn't be touched within the scope of fixing CVEs in DDF.

Reply:
Might be a good idea to update the commit message to say that the update in only in tika-bundle.

Reply:
Good call - can do. I'll likely take this PR down and do 2.10.1 if everyone agrees that's the route, so I can update the message for the next PR. Thanks for calling it out.

Reply:
I think you should double check that 2.9.10 still has a few CVEs that can't be suppressed before you go through the trouble again of trying to upgrade to 2.10.1.